packages/opentelemetry-instrumentation/test/common/Instrumentation.test.ts

```diff
@@ -56,4 +57,62 @@ describe('BaseInstrumentation', () => {
       assert.strictEqual(called, true);
     });
   });
+
+  describe('_onRequire', () => {
```

Comment:
it should not be added here but in node test only. Common is run for browser and for node

Reply:
This explains the CI failure. The browser tests won't run on my machine. I'll move these tests. Apologies for being new, appreciate all the help.

Reply:
No need to apology, we are here to help :), in tests you have folders: `browser`, `node`, `common`, just move your tests to folder `node`. This way your tests will be run only in node environment